### PR TITLE
2 documentation

### PR DESCRIPTION
**Pull request by valakis**

### CODE REVIEW

File: lib/src/components/drawer/drawer.dart [hunk 1]
```diff
@@ -3,7 +3,18 @@ import 'package:angular_components/src/model/ui/toggle.dart';
 
 import '../sidebar/sidebar.dart';
 
-/// Drawer is such a sidebar
+/// A Drawer is such a sidebar. (See more at)[https://material.io/guidelines/patterns/navigation-drawer.html#]
```
Comment (daniel-v) [inline lib/src/components/drawer/drawer.dart]: ... much awesome, wow! 🐕 

Should probably update that description. Maybe
```
Drawer serves as a navigational side panel.
```

File: lib/src/components/infobar/infobar.dart [hunk 2]
```diff
@@ -4,15 +4,22 @@ import 'package:angular_components/src/components/glyph/glyph.dart';
 import 'package:angular_components/src/components/material_button/material_button.dart';
 import 'package:angular_components/src/utils/async/async.dart' show LazyEventEmitter;
 
+/// An Inforbar is compositing an (icon as button)[https://material.io/components/web/catalog/buttons/icon-toggle-buttons/] and
+/// an arbitrary component.
```
Comment (daniel-v) [inline lib/src/components/infobar/infobar.dart]: `textual content`

Comment (daniel-v) [inline lib/src/components/infobar/infobar.dart]: Should also add a note:

```
Infobar is designed to display small notifications, important messages to the user, *in-context*. This differs from "snackbars" and "toasts" that serve as feedback of actions.
```

File: lib/src/components/sidebar/sidebar.dart [hunk 3]
```diff
@@ -1,6 +1,19 @@
 import 'package:angular2/core.dart';
 import 'package:angular_components/src/model/ui/toggle.dart';
 
+/// Sidebar. [See more at](https://material.io/guidelines/layout/structure.html#structure-side-nav)
```
Comment (daniel-v) [inline lib/src/components/sidebar/sidebar.dart]: Also add a note and open an issue with sidebar roadmap

```
*Note:* Sidebar is not yet fully implemented. Currently only support use within a Drawer. Please see [#123](plans for sidebar)
```

File: lib/src/components/sidebar_item/sidebar_item.dart [hunk 4]
```diff
@@ -2,7 +2,8 @@ import 'package:angular2/angular2.dart';
 import 'package:angular_components/src/components/glyph/glyph.dart';
 import '../../util/attribute.dart' as attrib;
 
-/// Toolbar Item
+/// Toolbar item. See more [at about normal lists](https://material.io/guidelines/components/lists.html#) or
```
Comment (daniel-v) [inline lib/src/components/sidebar_item/sidebar_item.dart]: You don't need `at`

File: lib/src/components/card/card.dart [hunk 5]
```diff
@@ -73,15 +83,19 @@ class SkawaCardContentComponent {
...
 ///
 ///     <skawa-card>
 ///       <skawa-card-header statusColor="rgb(255,0,0)">
-///
 ///       </skawa-card-header>
 ///     </skawa-card>
+///
+/// __Inputs:__
+///   - `statusColor: String` -- The color of the box-shadow-top. Accept only rgb() or rgba() formats. Defaults to transparent.
+///   - `backgroundColor: String` -- ???
```
Comment (daniel-v) [inline lib/src/components/card/card.dart]: Missing documentation line. We can probably remove this input unless we find a good use-case to support it.

File: lib/src/components/drawer/drawer.dart (hunk 1 above)
Comment (valakis) [inline lib/src/components/drawer/drawer.dart]: Done.

File: lib/src/components/infobar/infobar.dart (hunk 2 above)
Comment (valakis) [inline lib/src/components/infobar/infobar.dart]: Done.

File: lib/src/components/sidebar/sidebar.dart (hunk 3 above)
Comment (valakis) [inline lib/src/components/sidebar/sidebar.dart]: Done.

File: lib/src/components/sidebar_item/sidebar_item.dart (hunk 4 above)
Comment (valakis) [inline lib/src/components/sidebar_item/sidebar_item.dart]: Done.

File: lib/src/components/card/card.dart (hunk 5 above)
Comment (valakis) [inline lib/src/components/card/card.dart]: Done.

**Pull request merged**